tailcfg/tailcfg.go

```diff
@@ -2079,8 +2098,8 @@ type MapResponse struct {
 }
 
 // DisplayMessage represents a health state of the node from the control plane's
-// perspective. It is deliberately similar to health.Warnable as both get
-// converted into health.UnhealthyState to be sent to the GUI.
+// perspective. It is deliberately similar to [health.Warnable] as both get
+// converted into [health.UnhealthyState] to be sent to the GUI.
 type DisplayMessage struct {
 	// Title is a string that the GUI uses as title for this message. The title
```

Comment:
Can the title be the same as Text? Does Title need to end with a period? Can it start with a lowercase symbol?

Reply:
I agree we should document this, but I'm not sure this code is the right place? E.g. maybe it should be in tailcfgout. I'd also want to get product/design involved before codifying guidelines. In short: can this wait or is it a blocker for this PR?

Reply:
I think this file is probably the right place, since it allows folks like Headscale to understand the context and align with what we're doing. My preference would be to document our current understanding, and change later if we reconsider. A comment does not really commit us to anything long term? We could make it explicit that this stuff is being built out and can change.

(I don't feel too strongly about this, so it's up to you - not a blocker from my pov)

Reply:
👍 I've pieced together some basic guidance based on how I see these fields used in practice in existing Warnables, and also pointed the reader to look at those for more examples.